CONTRIBUTING.md

```diff
@@ -4,13 +4,23 @@ A document with pragmatic standards and best-practices used by Nation3 developer
 
 ## Pull Requests
 
-If the pull request is part of a retroactive [Dework](https://app.dework.xyz/nation3) session, remember to include a link to a Dework task in the description:
+### Pull Request Philosophy
+
+Pull requests should always be focused. For example, a pull request could add a feature, fix a bug, or refactor code; **but not a mixture**. Please also avoid super pull requests which attempt to do too much, are overly large, or overly complex as this makes review difficult.
```

Comment:
I have mixed fealings about this. Yes, in the optimal scenario for projects in production each pull request must serve a purpose and only one purpose.

But, when a project is still in development is very common that a PR includes multiple changes at once, as most probably a new feature or a fix can easily lead to a refactor, or even multiple features are changed at the same time.

As we have a tipicall review time of 24-48 hours with most PRs we shouldn't let this restriction to slow or block the developer workflow when the "mixture PR" is justified.

For maintenance and improvement of already ongoing projects I agree we should stick to incremental improvements. Hopefully the review time can be reduced with the increase in availability of active contributors in the future.

Reply:
As a "Commit Philosophy" I totally agree with your proposal

Reply:
@xPi2 Done:  https://github.com/nation3/.github/commit/f53b320f152ea088597c622c8a1fb9b6a21c9640